README.md

```diff
@@ -103,246 +103,10 @@ file has been signed by an authorized member of the Node.js team.
 Once verified, use the SHASUMS256.txt.asc file to get the checksum for
 the binary verification command above.
 
-## Build
+## Building nodejs
 
-### Unix / Macintosh
+See [BUILDING.md](BUILDING.md) for further instructions on how to build nodejs.
```

Comment:
Nit: `for further instructions` -> `for instructions`

Nit: `nodejs` -> `Node.js`

Tentative nit: `how to build Node.js` -> `how to build Node.js from source`